structured_type_test.go

```diff
@@ -236,19 +237,26 @@ func TestArrayAndMetadata(t *testing.T) {
 					actual:    []float64{},
 				},
 				{
-					name:      "number - fixed",
+					name:      "number - fixed integer",
 					query:     "SELECT ARRAY_CONSTRUCT(1, 2)::ARRAY(NUMBER(38, 0)) as structured_type UNION SELECT ARRAY_CONSTRUCT(3)::ARRAY(NUMBER(38, 0))",
 					expected1: []int64{1, 2},
 					expected2: []int64{3},
 					actual:    []int64{},
 				},
 				{
-					name:      "number - real",
+					name:      "number - fixed fraction",
 					query:     "SELECT ARRAY_CONSTRUCT(1.1, 2.2)::ARRAY(NUMBER(38, 19)) as structured_type UNION SELECT ARRAY_CONSTRUCT()::ARRAY(NUMBER(38, 19))",
 					expected1: []float64{1.1, 2.2},
 					expected2: []float64{},
 					actual:    []float64{},
 				},
+				{
+					name:      "double",
+					query:     "SELECT ARRAY_CONSTRUCT(1.1, 2.2)::ARRAY(DOUBLE) as structured_type UNION SELECT ARRAY_CONSTRUCT()::ARRAY(DOUBLE)",
+					expected1: []float64{1.1, 2.2},
+					expected2: []float64{},
```

Comment:
replacing `expected1` and `expected2` with an array of expected values would benefit this test with more flexibility of assertions. Same with `TestMapAndMetadata`

Reply:
To be honest I don't understand what you mean. You mean renaming the variable or what?

Reply:
I meant having an array instead of two fields. Just cause of scalability instead of potential introducing expected3 ;)

Reply:
Hmmm, I'd like to leave it as is, as it is easier to read, but if you insist we can change it :)

Reply:
Ok, if you say so.